web/src/infra/i18n/hocs/withTranslate.tsx

```diff
@@ -0,0 +1,35 @@
+import { getDisplayName } from 'recompose';
+import { nextI18Next } from '../config';
+import React from 'react';
+import { useCurrentGame } from 'infra/game/GameProvider';
+
+export const withTranslate = (propertyName: string = 'translate', options: { withRef?: boolean } = {}) => {
+  return function Extend(WrappedComponent) {
+    function I18nextWithTranslation({ forwardedRef, ...rest }) {
+      const { game } = useCurrentGame();
+      const ns = game?.code;
```

Comment:
NIT I think it would be more readable if you use more descriptive variables, i.e. "namespace" instead of "ns".

Reply:
Great suggestion!